### PR TITLE
[TASK] Add AddionalConfiguration content at installation

### PR DESCRIPTION
**Pull request by xf-**
@NamelessCoder please take a look :)

### CODE REVIEW

**Comment by NamelessCoder** [comment]
Looks very good. Although... I think we need to somehow mark the methods on UpdateService that are there because the `ext_update` class requires them. E.g. `access()`, `main()` etc. Since we now remove them from the context of this legacy, interface-less class there's no longer anything to indicate which methods must be there - and some of the methods are obvious candidates for removal if somebody doesn't know about the `ext_update` class.

Btw... is there a new API to replace `ext_update`? If so, we really should use that and this PR would be an excellent opportunity!

**Comment by xf-** [comment]
I don't know a new way. Thats why i extend the service and it works :)
I put everything in a service to access it form everywhere, because ext_update can't be trusted, because it isn't unique (no Namespace and can exists in multiple extensions).

File: Classes/Service/UpdateService.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+		if (0 === count($lines) || '<?php' !== $lines[0]) {
+			array_unshift($lines, '<?php');
+		}
+		return $lines;
+	}
+
+	/**
+	 * Returns TRUE if either of the expected configuration lines
+	 * do not currently exist. If both exist, returns FALSE
+	 * meaning "no need to run the script"
```
Comment (NamelessCoder) [inline Classes/Service/UpdateService.php]: Please add line after this doc: `Is required by the extension manager, do not remove or make protected` and an `@api` annotation below.

File: Classes/Service/UpdateService.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+				return TRUE;
+			}
+		}
+		return FALSE;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function main() {
```
Comment (NamelessCoder) [inline Classes/Service/UpdateService.php]: Please add line after this doc: `Is required by the extension manager, do not remove or make protected` and an `@api` annotation below.

**Pull request merged**